templates/aws/index.html

```diff
@@ -0,0 +1,143 @@
+          <li class="p-list__item is-ticked">Updates mirrored and images published in all AWS regions worldwide</li>
+          <li class="p-list__item is-ticked">5-year lifetime</li>
+        </ul>
+        <p>Run Ubuntu 18.04 LTS: <a href="https://aws.amazon.com/marketplace/pp/B07CQ33QKV" class="p-link--external">Standard</a> | <a href="https://aws.amazon.com/marketplace/pp/B07J5RRYGN" class="p-link--external">Minimal</a></p>
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Ubuntu Pro for AWS</h3>
+      <p class="p-heading--two">Starting at 0.1c/hour</p>
+      <hr class="u-sv1">
+      <div class="p-card__content">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Includes all Ubuntu optimisations and updates of the free version</li>
+          <li class="p-list__item is-ticked">Expanded security covering application components delivered with Ubuntu</li>
+          <li class="p-list__item is-ticked">Kernel live patching for instant security and longer uptimes</li>
+          <li class="p-list__item is-ticked">FIPS 140-2 and CC-EAL certified components</li>
+          <li class="p-list__item is-ticked">Integration with AWS security and compliance features</li>
+          <li class="p-list__item is-ticked">10-year lifetime</li>
+        </ul>
+        <p><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-button--positive"><span class="p-link--external">Launch Ubuntu Pro 18.04 LTS</span></a></p>
+        <p><a href="/aws/pro">More about Ubuntu Pro for AWS&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="u-fixed-width">
+    <h2>Features</h2>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Optimised</h3>
+      <p>Ubuntu on AWS runs on an AWS-optimised kernel, which includes improved device drivers, like ENA, and out of the box support for accelerators like GPGPUs. This means faster instance starts and better runtime performance for your workloads.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Security built in</h3>
+      <p>Our security teams tracks alerts 24x7 and releases patches to system components and applications continuously. Canonical operates a worldwide distribution network ensuring that package updates are delivered in-region quickly.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">All the bits you need</h3>
+      <p>Ubuntu includes everything you need out of the box to build and operate cloud-native applications at scale. Supporting runtimes like Python, node.js and Go, and server components like Apache Kafka, NGINX, MongoDB, Redis and PostgreSQL.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```

Comment:
Don't think this needs to be a `row`
```suggestion
  <div class="u-fixed-width">
```

Reply:
Same as below